aurelia-slickgrid/src/aurelia-slickgrid/editors/singleSelectEditor.ts

```diff
@@ -122,23 +121,22 @@ export class SingleSelectEditor implements Editor {
   }
 
   private buildTemplateHtmlString() {
-    if (!this.columnDef || !this.columnDef.filter || !this.columnDef.filter.collection) {
-      throw new Error('[Aurelia-SlickGrid] You need to pass a "collection" for ' +
-        'the MultipleSelect Filter to work correctly. Also each option should include ' +
-        'a value/label pair (or value/labelKey when using Locale). For example:: ' +
-        '{ filter: type: FilterType.multipleSelect, collection: [{ value: true, label: \'True\' }, ' +
-        '{ value: false, label: \'False\'}] }');
+    if (!this.columnDef || !this.columnDef.params || !this.columnDef.params.collection) {
+      throw new Error('[Aurelia-SlickGrid] You need to pass a "collection" on the params property in the column definition for ' +
+        'the SingleSelect Editor to work correctly. Also each option should include ' +
+        'a value/label pair (or value/labelKey when using Locale). For example: { params: { ' +
+        '{ collection: [{ value: true, label: \'True\' }, { value: false, label: \'False\'}] } } }');
     }
-    this.optionCollection = this.columnDef.filter.collection || [];
-    this.labelName = (this.columnDef.filter.customStructure) ? this.columnDef.filter.customStructure.label : 'label';
-    this.valueName = (this.columnDef.filter.customStructure) ? this.columnDef.filter.customStructure.value : 'value';
+    this.collection = this.columnDef.params.collection || [];
+    this.labelName = (this.columnDef.params.customStructure) ? this.columnDef.params.customStructure.label : 'label';
+    this.valueName = (this.columnDef.params.customStructure) ? this.columnDef.params.customStructure.value : 'value';
 
     let options = '';
-    this.optionCollection.forEach((option: SelectOption) => {
+    this.collection.forEach((option: SelectOption) => {
       if (!option || (option[this.labelName] === undefined && option.labelKey === undefined)) {
         throw new Error('A collection with value/label (or value/labelKey when using ' +
-          'Locale) is required to populate the Select list, for example:: { filter: ' +
-          'type: FilterType.multipleSelect, collection: [ { value: \'1\', label: \'One\' } ])');
+          'Locale) is required to populate the Select list, for example: ' +
+          '{ collection: [ { value: \'1\', label: \'One\' } ])');
```

Comment:
You added an open curly brace in your throw error, though I think it's missing an ending curly brace.